File: verify.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('file://' + __dirname + '/index.html');
  await page.waitForTimeout(3500); // Wait for animation
  await page.screenshot({ path: 'verify_blob.png' });
  await browser.close();
})();

File: verify.py
from playwright.sync_api import sync_playwright
import os

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto('file://' + os.path.dirname(os.path.abspath(__file__)) + '/index.html')
    page.wait_for_timeout(3500)
    page.screenshot(path="verify_blob.png")
    browser.close()
